src/inverseRobot.js

```diff
@@ -7,7 +7,19 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const robotKeys = Object.keys(robot);
+  const robotValues = Object.values(robot);
+  const inversedRobotObj = {};
+
+  for (let i = 0; i < robotKeys.length; i++) {
```

Comment:
maybe, in this case, it might be better to use a loop "for in" for a cleaner and more readable code.

Reply:
Yep, I fully agree with you. Thanks a lot for your comment!

Reply:
I have changed my code a little bit